catalogs/bnf.yaml

```diff
@@ -10,7 +10,7 @@ sources:
       collection_url: https://gallica.bnf.fr/SRU?operation=searchRetrieve&version=1.2&query=%28%28bibliotheque%20adj%20%22Institut%20dominicain%20d%27%C3%A9tudes%20orientales%22%29%29#resultat-id-3
     metadata:
       data_path: bnf/ideo
-      config: bnf_config
+      config: bnf
```

Comment:
this should be `bnf_ideo`

Reply:
I expect bnf to have only one config as well but I may need to update this once I get into the traject migration